ooiui/static/css/common/toc_menu.css

```diff
@@ -2,8 +2,14 @@
     max-height:80%;
 }
 #assetBrowser {
-    margin-top:25px;
+    margin-top:5px;
 }
+
+.toc-header{
```

Comment:
@birdage agan, always remember that not ALL pages use the TOC identically. *winky face*